### ISSUE TITLE
request-repos: crash on comment generation

### ISSUE THREAD

**Issue by farchord** [juhp/fbrnch#45]
```
Thank you for the reviewfbrnch: Prelude.head: empty list
CallStack (from HasCallStack):
  error, called at libraries/base/GHC/List.hs:1646:3 in base:GHC.List
  errorEmptyList, called at libraries/base/GHC/List.hs:85:11 in base:GHC.List
  badHead, called at libraries/base/GHC/List.hs:81:28 in base:GHC.List
  head, called at src/Cmd/RequestRepo.hs:100:19 in main:Cmd.RequestRepo
```
Do keep in mind, when I ran this, the review was approved, but sitting in the wrong status (Assigned instead of POST), is it possible that's why?

**Comment by juhp** [comment]
Thank for reporting this. Perhaps it failed to get the name of the user I think.

**Comment by juhp** [comment]
Apparently the bug in question was https://bugzilla.redhat.com/show_bug.cgi?id=2244247
where the reviewer only has an email address apparently.

**Comment by juhp** [comment]
Should be fixed in [1.3.3](https://hackage.haskell.org/package/fbrnch-1.3.3)
